lib/scss/components/news-vertical-teaser/_news-vertical-teaser.scss

```diff
@@ -1,14 +1,84 @@
 @charset "UTF-8";
 
-.su-news-vertical-teaser {
+// For all templates.
+.views-view-grid {
+  .flex-container {
+    .more-news-view {
+      @include grid-media-max('md') {
+        display: block;
+
+        .flex-4-of-12 {
+          max-width: 100%;
+        }
+      }
+    }
+  }
+}
+
+// News Cards added in page as a paragraph.
+.flex-container {
+  &.more-news-view {
```

Comment:
Is the double selector needed here? Can we reduce the nesting by 1 and just use the `.more-news-view` selector?